README.md

```diff
@@ -6,25 +6,23 @@
 [![Maintainability](https://api.codeclimate.com/v1/badges/11711f7444ef6ee2b8fa/maintainability)](https://codeclimate.com/github/atlp-rwanda/bravo-bn-fe/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/11711f7444ef6ee2b8fa/test_coverage)](https://codeclimate.com/github/atlp-rwanda/bravo-bn-fe/test_coverage)
 
+## Description
 
-## Installation
-
-```bash
-$ npm install
-```
+Barefoot Nomad is an application that will enable its "Company Nomads" book their travel and accommodation easily and conveniently across all the locations/centers where the company has its operations.
+ 
 
-## Running the app
 
-```bash
-# development
-$ npm run start
+## Installation
 
-# watch mode
-$ npm run start:dev
+1. Install [Node js]()
+2. To clone, run `git clone https://github.com/atlp-rwanda/bravo-bn-fe.git`
+3. `cd` into the project directory
+4. Run `npm install` in the terminal to install all dependencies
+5. To run the application in development mode, run `npm run start`<br />
+Then open [http://localhost:3000](http://localhost:3000) to view it in the browser.
```

Comment:
this line is not neccessary because after running `npm run start` browser will open automatically